src/server/server_dispatch.c

```
@@ -820,6 +820,65 @@ bool bc_torpedo_target_pos(i32 target_id, bc_vec3_t *out_pos,
     return true;
 }
 
+/* Handle inbound owner StateUpdate (0x1C) as server input only.
+ * Downstream 0x1C stream is generated server-side in main.c. */
+static void handle_state_update_input(int peer_slot, bc_peer_t *peer,
+                                      const u8 *payload, int payload_len)
+{
+    if (g_registry_loaded && peer->has_ship) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)